Tasks/AzureTestPlanV0/utils.ts

```diff
@@ -21,3 +21,22 @@ export function replaceLastDotWithHash(inputString) {
     }
 }
 
+export function separatePath(inputString) {
+    const lastDotIndex = inputString.lastIndexOf('.');
+
+    if (lastDotIndex !== -1) {
+        const stringWith = inputString.slice(0, lastDotIndex);
+        return stringWith;
+    } else {
+        // If there is no dot in the string, return the original string
+        return inputString;
+    }
+}
+export function separateTestName(inputString) {
+    const lastDotIndex = inputString.lastIndexOf('.');
+
+    if (lastDotIndex !== -1) {
+        const stringWith =  inputString.slice(lastDotIndex+1) ;
+        return stringWith;
+    } 
```

Comment:
Should we have a default return statement if in case we dont enter in the IF statement?

Reply:
Sure, I will add it.